airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/assets.json

```
@@ -0,0 +1,10 @@
+{
+  "columns": {
+    "consumingDags": "消費Dag",
```

```suggestion
    "consumingDags": "消費者 Dag",
```

Basically, `consumingDags` are the dags that consume this asset event, and the downstream ones are consumers. I think it might be better to understand a bit 🤔

---

Sure, makes perfect sense!